WebRoot/js/zimbraMail/abook/view/ZmContactPicker.js

```diff
@@ -591,6 +587,16 @@ function(account) {
 	this._keyPressCallback = new AjxCallback(this, this._searchButtonListener);
     this.sharedContactGroups = [];
 
+	var habContainer = document.getElementById(this._htmlElId + "_habTree");
+	if (habContainer && this.sourceTreeView) {
+		var innerOverviewDiv = habContainer.getElementsByClassName("ZmOverview")[0];
+		Dwt.setSize(innerOverviewDiv, 'auto', habContainer.clientHeight);
+		Dwt.setScrollStyle(innerOverviewDiv, Dwt.SCROLL);
```

Comment:
isn't this needed in case of non-HAB as well?

Reply:
I think we should avoid application wide changes in this ticket. I recommends separate ticket.
https://jira.corp.synacor.com/browse/ZBUG-920